fxa-rust-client/src/lib.rs

```diff
@@ -50,6 +50,21 @@ impl FxAConfig {
     }
 }
 
+#[derive(Serialize, Deserialize)]
+struct FxAStateV1 {
+    config: FxAConfig,
+    uid: String,
+    email: String,
+    login_state: FxaLoginState,
```

Comment:
nit: can we be consistent with "FxA" vs. "Fxa" please?  Neither are particularly Rust friendly, which prefers snake case for method names.  Can we drop FxA entirely except for a few top-level types?  We have module scoping.